components/ui/AppMegaDropdownMenu.vue

```diff
@@ -1,38 +1,63 @@
 <template>
   <article class="app-mega-dropdown">
-    <button
-      ref="button"
-      class="app-mega-dropdown__button"
-      :class="`app-mega-dropdown__button_${kind}`"
-      @click="toggleOpen"
+    <label
+      ref="searchFieldWrapper"
+      class="app-mega-dropdown__search-field-wrapper"
+      :class="`app-mega-dropdown__search-field-wrapper_${kind}`"
     >
-      <span>{{ placeholder }}</span>
-      <svg class="app-mega-dropdown__icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 32 32"><path d="M16 22L6 12l1.4-1.4 8.6 8.6 8.6-8.6L26 12z" /></svg>
-    </button>
+      <input
+        v-model="textOnTheFilter"
```

Comment:
We could simplify the name of this variable:

```suggestion
        v-model="filterText"
```

Note: We would also need to update it in the lines 89, 92, 96 and 228.

Reply:
I'm not agree with this simplification: https://github.com/Qiskit/qiskit.org/pull/1823#discussion_r638051111. what do you think?

Reply:
Good point. But I think that possible confusion might happen only if it's a function:

* Function: `filterText` seems to mean "please filter this text"
* Variable: `filterText` seems to mean "the text of the filter"
* Variable: `filteredText` seems to mean "the text that was filtered"

Having said that, this is just my way of seen it, and the code should be clear for everybody, so I'm also OK with leaving it as is, too. 🙂

Reply:
You both are going to work more with the code than me, so if for you is more clear `filterText`, go ahead :)

Reply:
You decide @korgan00 😄

Reply:
I will leave it as is. If one person reads it better this way, it increases readablity.